client/lib/signup/step-actions/index.js

```diff
@@ -1256,6 +1256,7 @@ export function isNewOrExistingSiteFulfilled( stepName, defaultDependencies, nex
 		if ( stepProvidesDependency ) {
 			dependency = {
 				newOrExistingSiteChoice: 'new-site',
+				forceAutoGeneratedBlogName: true,
```

Comment:
Reference for this change: https://github.com/Automattic/wp-calypso/pull/60998#issuecomment-1036099100